sdk/core/azure-core/inc/azure/core/http/win_http_transport.hpp

```diff
@@ -21,7 +21,6 @@ namespace Azure { namespace Core { namespace Http {
   namespace _detail {
 
     constexpr static int64_t DefaultUploadChunkSize = 1024 * 64;
-    constexpr static int64_t MaximumUploadChunkSize = 1024 * 1024;
```

Comment:
We still want to keep this.

Reply:
what's the benefit? we are adding an extra check which we don't need